src/Stratis.FederationGatewayD/Program.cs

```diff
@@ -82,5 +83,23 @@ public static async Task RunFederationGatewayAsync(string[] args)
                 Console.WriteLine("There was a problem initializing the node. Details: '{0}'", ex.Message);
             }
         }
+
+        private static IFullNode GetFederatedPegFullNode(NodeSettings nodeSettings)

```

Comment:
Can you just replace this directly in line 64? No one is using it at the moment.

Reply:
Oh sorry I meant to use this method after writing it ! thanks